src/pages/Error404Page.vue

```diff
@@ -1,41 +1,208 @@
+                </h2>
+            </div>
         </div>
 
-        <q-btn
-            class="q-mt-xl"
-            color="white"
-            text-color="blue"
-            unelevated
-            to="/"
-            label="Go Home"
-            no-caps
-        />
+        <div class="c-404-page__layout-r3">
+            <div class="c-404-page__layout-r3-c1">
+                <p class="c-404-page__text">
+                    {{ $t('error404.text') }}
+                </p>
+            </div>
+        </div>
+
+        <div class="c-404-page__layout-r4">
+            <div class="c-404-page__layout-r4-c1">
+                <q-btn color="primary" to="/" class="c-404-page__button">
+                    {{ $t('error404.take_me_to_home') }}
+                </q-btn>
+            </div>
+        </div>
     </div>
+
 </div>
 </template>
 
 <style lang="scss" scoped>
-.main-text {
-    font-size: 30vh;
-}
 
-.oops {
-    opacity: 0.4;
+.c-404-page {
+
+    &__video-wrapper {
+        background-color: black;
+        position: absolute;
+        overflow: hidden;
+        z-index: -1;
+        top: 0px;
+        bottom: 0px;
+        left: 0px;
+        right: 0px;
+    }
+
+    &__video {
+        object-fit: cover;
+        width: 100vw;
+        height: 100vh;
+        position: absolute;
+        transform: rotate(180deg);
+        top: 35vh;
+        left: 0;
+    }
+
+    &__video-overlay {
+        background: linear-gradient(0.4turn, #0a1d5f52, #814cdc52);
+        object-fit: cover;
+        position: absolute;
+        overflow: hidden;
+        z-index: -1;
+        top: 0px;
+        bottom: 0px;
+        left: 0px;
+        right: 0px;
+    }
+
+    // layout
+
+    &__layout {
+        max-width: 550px;
+        width: 100vw;
+        height: 100vh;
+        display: flex;
+        flex-direction: column;
+        justify-content: initial;
+        margin-top: 0vh;
+
+        @media only screen and (min-width: $breakpoint-md-min) {
+            margin-top: 15vh;
+        }
+    }
+
+    &__layout-r1 {
+        display: flex;
+        justify-content: center;
+    }
+
+    &__layout-logo {
+        display: flex;
+    }
+
+    &__layout-title {
+        text-align: left;
+        margin-left: 25px;
+    }
+
+    &__logo {
+        width: 100%;
+        max-width: 115px;
+        .landscape & {
+            max-width: 80px;
+        }
+    }
+
+    &__title {
+        font-size: 73.61px;
+        font-weight: 400;
+        color: white;
+        .landscape & {
+            font-size: 48px;
+        }
+    }
+
+    &__subtitle {
+        font-size: 24px;
+        font-weight: 600;
+        color: white;
+        line-height: 1.95rem;
+        .landscape & {
+            font-size: 20px;
+            margin: 5px 0 0 0;
+        }
+    }
+
+    &__text {
+        font-size: 16px;
+        font-weight: 400;
+        color: white;
+    }
+
+    &__button {
+        --q-primary: linear-gradient(106.63deg, #1AD6FF 0%, #8946DF 83.87%);
```

Comment:
i think you can just use the class `purpleGradient` on the button itself and remove this style, like the homepage button

Reply:
Oh, Thanks! I didn't know that